VirtualAssistantBusinessLogic/VirtualAssistantBusinessLogic/SparQL/SparQLBuilder.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VirtualAssistantBusinessLogic.KnowledgeGraph;
+
+namespace VirtualAssistantBusinessLogic.SparQL
+{
+    /// <summary>
+    /// Abstract class for SparQL Builders
+    /// </summary>
+    public abstract class SparQLBuilder : ISparQLBuilder
+    {
+        public SparQLBuilder(ISPOEncoder spoEncoder)
+        {
+            SPOEncoder = spoEncoder;
+        }
+
+        public string Query { get; set; }
+        protected ISPOEncoder SPOEncoder { get; set; }
+
+        public string[] WantedProperties { get; set; }
```

Comment:
Maybe move variable to between line 14 and 15?

Reply:
Normalt har man constructor øverst og så properties og fields under